src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -112,6 +112,12 @@ abstract class AbstractNormalizer implements NormalizerInterface, DenormalizerIn
      */
     public const IGNORED_ATTRIBUTES = 'ignored_attributes';
 
+    /**
+     * Flag to prevent fields that are nullable to receive a null
+     * fallback value when no parameters are provided for it.
```

Comment:
Require all properties to be listed in the input instead of falling back to null for nullable ones.